docs/data-tunnel/03-tunnel-creation.md

```diff
@@ -3,41 +3,57 @@
 
-| Field | Description |
-| -- | - |
-| signal_id | The signal id (symbol) to feed price |
-| deviation_bps | Addtional price deviation threshold (in basis point) |
-| interval | The default feeding interval in seconds |
+| Field         | Description                                           |
+| ------------- | ----------------------------------------------------- |
+| signal_id     | The signal id (symbol) to feed price                  |
+| deviation_bps | Additional price deviation threshold (in basis point) |
+| interval      | The default feeding interval in seconds               |
 
-## Tunnel Route and Feed Type
+## Tunnel Route
 
-The creator also specifies the **route** to deliver price data to the destination, currently [TSS](./05-tss-integration.md), [IBC](./06-ibc-intergation.md), and [Axelar](./07-axelar-integration.md) routes are supported. Additionally, the creator specifies the encoding scheme to encode price data as `feed_type`. The current supported values are fixed point decimal encoding and tick encoding.
+The creator also specifies the **route** to deliver price data to the destination, currently [TSS](./06-tss-integration.md), [IBC](./07-ibc-intergation.md), [IBC Hook](./08-ibc-hook-integration.md), and [Router](./09-router-integration.md) routes are supported.
 
 ## Initial Deposit
 
-Lastly, the creator must seed the initial BAND token deposit to the tunnel. This allows the tunnel to instantly be ready to operate the moment it is created. Once a tunnel is created, a dedicated BandChain address will be created and anyone can send BAND token to the address to fund the tunnel. As specified in [the previous section](./02-tunnel-architecture.md), every message sent from a tunnel will incur various fees. If the tunnel runs out of BAND token, it will temporarily stop the operation until more BAND is sent and the creator broadcast `MsgActivateTunnel`.
+The creator must seed the initial BAND token deposit to the tunnel. This initial deposit ensures that the tunnel is ready to operate immediately upon creation, provided it meets the minimum deposit requirements specified in the Tunnel Params. If the tunnel's balance falls below the required amount, others can send deposits to the tunnel by broadcasting a `MsgDepositTunnel` message to increase the total deposit of the tunnel by `TunnelID`.
+
+## Fee Payer Funds
+
+Lastly, once a tunnel is created, a dedicated BandChain address will be generated as the fee payer. Anyone can send BAND tokens to this address to fund the tunnel for operations. As specified in [the previous section](./02-tunnel-architecture.md), every message sent from a tunnel will incur various fees. If the tunnel runs out of BAND tokens, it will temporarily stop operating until more BAND tokens are sent and the creator broadcasts a `MsgActivateTunnel` message.
```

Comment:
`As specified in [the previous section](./02-tunnel-architecture.md), every message sent from a tunnel will incur various fees`. <<< can't find this detail in the previous section.